src/js/components/Calendar/__tests__/Calendar-test.js

```diff
@@ -149,4 +164,297 @@ describe('Calendar', () => {
     expect(onSelect).toBeCalledWith(expect.stringMatching(/^2018-01-17T/));
     expect(container.firstChild).toMatchSnapshot();
   });
+
+  test('first day sunday week monday', () => {
+    // the first day of the month is on Sunday but the
+    // user wants the week to start on Monday
+    const { container } = render(
+      <Grommet>
+        <Calendar
+          firstDayOfWeek={1}
+          date="2020-03-01T00:00:00-08:00"
+          animate={false}
+        />
```

Comment:
Is this the same as `<Calendar firstDayOfWeek={1} date={new Date(2019, 8, 2).toISOString()} />` as appears on storybook? If so, then we should remove that story from storybook and only keep the unit test.

We can add the comment from storybook to this unit test:
>// When the first day of the month is Sunday, and the request of firstDayOfWeek
>// is Monday, we are verifing we are not missing a week, issue 3253.

Reply:
Yes, it is the same as the storybook example. Should I remove the storybook example in this PR or open a new PR to do that

Reply:
In this PR please.

Reply:
I looked though storybook and for Calendar there aren't any other examples showing firstDayOfWeek so I will leave the example in storybook as discussed